sdk/selfhelp/arm-selfhelp/CHANGELOG.md

```diff
@@ -1,14 +1,97 @@
 # Release History
+    
+## 2.0.0-beta.2 (2023-12-04)
+    
+**Features**
 
-## 2.0.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+  - Added operation group CheckNameAvailability
```

Comment:
I didn't see any api version change, could you point this for me?

Reply:
the version of this release request is the same as last release request.